python/ql/test/library-tests/frameworks/tornado/taint_test.py

```diff
@@ -3,73 +3,73 @@
 
 class TaintTest(tornado.web.RequestHandler):
     def get(self, name = "World!", number="0", foo="foo"):  # $ requestHandler routedParameter=name routedParameter=number
-        ensure_tainted(name, number)
+        ensure_tainted(name, number) # $ tainted
```

Comment:
Here you now have to split into two lines.

Reply:
no, this just means that all arguments on that line are tainted :wink: